src/pages/index.tsx

```diff
@@ -39,10 +39,10 @@ const links = [
         color: '#1099A8',
     },
     {
-        text: 'XP Table',
-        url: '/xp',
+        text: 'Epic Spells Things',
+        url: '/epic',
         badge: true,
-        description: "Here's the 5e XP table because no one ever knows these numbers",
+        description: "Knowing magic is cool This'll make you cooler.",
         color: '#1099A8',
     },
```

Comment:
Are we getting rid of the XP table?

Reply:
I guess I preemptively pulled this. We had talked briefly about narrowing scope of things and I think past Zander just pulled this based on that conversation. The page still exists. I was just removing the link. We can keep it 👍

Reply:
b5e8152